frontend/src/App.tsx

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable react/jsx-no-undef */
+import { useQuery, gql, DocumentNode } from "@apollo/client";
+import React from "react";
+import { theme } from "@diamondlightsource/ui-components"
+import { ChakraProvider } from "@chakra-ui/react";
+import { LoadMoreTable } from "./components/LoadMoreTable";
+
+const GET_INFO: DocumentNode = gql`
+query pinInfo ($after: String) {
+  libraryPins(first: 2, after: $after) {
+    pageInfo {
+      hasPreviousPage,
+      hasNextPage,
+      startCursor,
+      endCursor
+    },
+    edges {
+      cursor
+      node {
+        barcode,
+        loopSize,
+        status
+      }
+    }
+  }
+}
+`;
+
+function DisplayPinInfo(): React.JSX.Element {
```

Comment:
We want a docstring for this function, and probably a lint to ensure we don't forget (happy for that to be made into an issue though)